docs/open-source-project/porting/system.md

```diff
@@ -100,6 +100,14 @@ To use most of the `resourced` functionalities, you must enable the following `c
 >
 > To use the `resourced` freezer feature, you must install the freezer plugin by enabling `CONFIG_FREEZER`.
 
+Resourced supports several swapping scenarios to provide more RAM to applications running in the foreground. To support them, enable the following options in the kernel configuration:
```

Comment:
Please rephrase to:

`resourced` supports multiple swapping scenarios to provide more RAM to the applications running in the foreground. To support the applications, enable the following options in the kernel configuration:

Question:  Please confirm, does "them" in the original sentence refer to "the running applications"?

Reply:
No, them refers to "scenarios".